src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -94,28 +102,112 @@ following to integrate with your `my_flutter` app:
 
 3. Run `pod install`.
 
+{{site.alert.note}}
 Whenever you change the Flutter plugin dependencies in `some/path/my_flutter/pubspec.yaml`,
 you need to run `flutter pub get` from `some/path/my_flutter` to refresh the list
 of plugins read by the `podhelper.rb` script. Then run `pod install` again from
 `some/path/MyApp`.
+{{site.alert.end}}
 
-The `podhelper.rb` script ensures that your plugins, the Flutter.framework, and
-the App.framework are embedded in your project.
+The `podhelper.rb` script ensures that your plugins, `Flutter.framework`, and
+`App.framework` are embedded in your project.
 
 You should now be able to build the project using `⌘B`.
 
 ### Under the hood
 
-If you have some reason to do this manually or debug why these steps aren't
-working, here's what's going on under the hood:
-
-- `Flutter.framework` (the Engine library) is getting embedded into your app for you.  This has to match up with the release type
-(debug/profile/release) as well as the architecture for your app (arm*, x86_64, etc.).  CocoaPods pulls this in as a vendored
-framework and makes sure it gets embedded into your native app.
-- `App.framework` (your Flutter application binary) is embedded into your app.  CocoaPods also pulls this in as a vendored framework and
-makes sure it gets embedded into your native app.
-- Any plugins are added as CocoaPod pods.  In theory, it should be possible to manually merge those in as well, but those instructions
-vary on the pod dependencies of each plugin.
-- A build script is added to the Podfile targets that call `install_all_flutter_pods` to ensure that the binaries you build stay up to date
-with the Dart code that's actually in the folder.  It also uses your Xcode build configuration (Debug, Profile, Release) to embed the matching
-release type of `Flutter.framework` and `App.framework`.
\ No newline at end of file
+- `Flutter.framework` (the Engine library) is being embedded into your application by
+CocoaPods as a vendored framework. This has to match up with the release type
+(debug/profile/release) as well as the architecture for your app (arm*, 
+x86_64, etc.).
+- `App.framework` (your Flutter application binary) is being embedded into your
```

Comment:
The Flutter framework is in there too

Reply:
Can you use "*" to indicate a bulleted list, rather than minus? (I know this was there before...)

Also, I am not sure what the point of this section is. It seems like it's there to just tell the user what's going on, but you've deleted the introductory sentence explaining that it's there just to let you know what's going on. Can you add that back (or something similar)?

Reply:
I don't know if it's useful, the contents are almost untouched from what was there before.  I'm happy to remove the entire section if you don't think it's useful.  I removed the intro because I didn't want to imply anything would go wrong following these steps.